regression9.py

```diff
@@ -0,0 +1,18 @@
+# create Fibonacci Recursion
+list=[]
+def FibRecursion(rnumb):
+    if rnumb == 1:
+        return 0
+    elif rnumb ==2:
+        return 1
+    else:
+        nfib = FibRecursion(rnumb-1)+FibRecursion(rnumb-2)
+        return nfib
+
+inpR = int(input("Enter integer for Fibonacci Recursion"))
+for i in range(inpR):
+    next= FibRecursion(i+1)
+    list.append(next)
+flist = FibRecursion(inpR)
```

Comment:
The result of your function is an integer, but you are trying to get an element from it, like from a list. This breaks your program

Reply:
Sorry, was experimenting and forgot to commit last version of this file. Now everything is fine